samples/src/test/java/org/autorefactor/refactoring/rules/samples_out/BooleanSample.java

```diff
@@ -278,7 +355,8 @@ public Boolean doNotThrowAnyException(boolean bo) {
             Boolean b;
         }
         ClassWithBooleanField objWithBooleanField = new ClassWithBooleanField();
-        return bo ? objWithBooleanField.b : Boolean.TRUE;
+        Boolean aBoolean = Boolean.TRUE;
+        return bo ? objWithBooleanField.b : aBoolean;

```

Comment:
Please put the replacement here.
I just wanted to be sure that there was no exception thrown here.